tests/integration/test_llmcache.py

```diff
@@ -130,3 +160,26 @@ def test_multiple_items(cache, vectorizer):
         print(check_result, flush=True)
         assert check_result[0]["response"] == expected_response
         assert "metadata" not in check_result[0]
+
+# Test retrieving underlying SearchIndex for the cache.
+def test_get_index(cache):
+    assert type(cache.index) is SearchIndex
```

Comment:
`assert isinstance(cache.index, SearchIndex)` is the preferred python check for this!

Reply:
Changed!